starboard/extension/loader_app_metrics.h

```diff
@@ -38,6 +38,16 @@ typedef enum CrashpadInstallationStatus {
   kMaxValue = kFailedSignalHandlerInstallationFailed
 } CrashpadInstallationStatus;
 
+typedef enum class SlotSelectionStatus {
```

Comment:
I think we might want to add more context or have more values.
For example we have different reasons to revert. It would be good to know what caused revert.
    starboard/loader_app/slot_management.cc